js/ui/status/missionGame.js

```diff
@@ -0,0 +1,528 @@
+                                           pseudo_class: 'highlighted' });
+    separator.connect('repaint', Lang.bind(this, function(area) {
+        let cr = area.get_context();
+        let themeNode = area.get_theme_node();
+        let [width, height] = area.get_surface_size();
+        let stippleColor = themeNode.get_color('-stipple-color');
+        let stippleWidth = themeNode.get_length('-stipple-width');
+        let x = Math.floor(width/2) + 0.5;
+        cr.moveTo(x, 0);
+        cr.lineTo(x, height);
+        Clutter.cairo_set_source_color(cr, stippleColor);
+        cr.setDash([1, 3], 1); // Hard-code for now
+        cr.setLineWidth(stippleWidth);
+        cr.stroke();
+        cr.$dispose();
+    }));
+
+    return separator;
+}
+
+/* Separate indicators option */
+const MissionGameToolboxIndicator = new Lang.Class({
+    Name: 'MissionGameToolboxIndicator',
+    Extends: PanelMenu.SystemStatusButton,
+
+    _init: function() {
+        this.parent('starred-symbolic', _("Mission"));
+        this.setIcon('starred-symbolic');
+
+        this._service = new MissionGameService.MissionChatboxTextService();
+
+        /* Create layout for indicator menu */
+        const hbox = new St.BoxLayout({name: 'menuArea'});
+
+        /* Add toolbox, separator, chatbox */
+        hbox.add(new MissionToolbox({}, this.menu, this._service));
+
+        this.menu.addActor(hbox);
+    }
+});
+
+
+const MissionGameChatboxIndicator = new Lang.Class({
+    Name: 'MissionGameChatboxIndicator',
+    Extends: PanelMenu.SystemStatusButton,
+
+    _init: function() {
+        this.parent('folder-drag-accept-symbolic', _("Mission"));
+        this.setIcon('folder-drag-accept-symbolic');
+
+        this._service = new MissionGameService.MissionChatboxTextService();
+
+        /* Create layout for indicator menu */
+        const hbox = new St.BoxLayout({name: 'menuArea'});
+
+        /* Add toolbox, separator, chatbox */
+        hbox.add(new MissionChatbox({}, this._service));
+        this.menu.addActor(hbox);
+
+        /* Tell the service to commence the intro lesson, though this will only do
+         * so if the service is in a state where the intro lesson has not yet been
+         * commenced. */
+        this.menu.connect('open-state-changed', Lang.bind(this, function(menu, state) {
+            if (state) {
+                this._service.commenceIntroLesson();
+            }
+        }));
+    }
+});
+
+
+/* Combined indicator option */
+const MissionGameIndicator = new Lang.Class({
+    Name: 'MissionGameIndicator',
+    Extends: PanelMenu.SystemStatusButton,
+
+    _init: function() {
+        this.parent('folder-drag-accept-symbolic', _("Mission"));
+        this.setIcon('folder-drag-accept-symbolic');
+
+        this._service = new MissionGameService.MissionChatboxTextService();
+
+        /* Create layout for indicator menu */
+        const hbox = new St.BoxLayout({name: 'menuArea'});
```

Comment:
`const` is only used for constants or imports in the shell coding style (many occurrences). In this case, typically you'd use `let`